docs/components/modeler/web-modeler/deploy-process-application.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: deploy-process-application
 title: Validate and deploy your process application
-description: You can deploy your process application to a cluster, and run your process application to test, debug, and observe how it performs in a live environment.
+description: Validate your process application in development, before deploying it to testing, staging, or production.
```

Comment:
Grammar?

Reply:
I went back and forth on this comma! I eventually kept it in as I wanted to make the user pause before the second concept (deployment) - without it, it felt like there was too much to take in in one go - does that make sense?

Reply:
Changed in the end as suggested, perhaps having the comma doesn't work after all 🤷 - thank you!